scripts/update-icons.js

```diff
@@ -3,19 +3,19 @@
  This script extracts SVG icons from Skin and saves them as React TSX
  */
 const { resolve } = require('path');
-const { saveIconType, saveIconConstants, saveSvgIcons, parseSVG, getIconKeys } = require('./update-icons-helpers');
+const { saveIconType, saveIconConstants, saveSvgIcons, parseSVG, parseSVGSymbols, getIconKeys } = require('./update-icons-helpers');
 
 const skinIconsFile = resolve(process.cwd(), 'node_modules/@ebay/skin/dist/svg/icons.svg')
 const typesFile = resolve(__dirname, `../src/ebay-icon/types.ts`)
 const constFile = resolve(__dirname, `../src/ebay-icon/__tests__/constants.ts`)
 const ebaySvgFile = resolve(__dirname, `../src/ebay-svg/svg.tsx`)
 
-const skinSVGSymbols = parseSVG(skinIconsFile);
+const skinSVG = parseSVG(skinIconsFile)
 
-console.log(`Found ${skinSVGSymbols.length} icons in Skin.`)
-
-saveSvgIcons(skinSVGSymbols, ebaySvgFile)
+saveSvgIcons(skinSVG, ebaySvgFile)
 
+const skinSVGSymbols = parseSVGSymbols(skinIconsFile)
+console.log(`Found ${skinSVGSymbols.length} icons in Skin.`)
```

Comment:
not part of this PR, but curious  - do we have any significance in this console log?

Reply:
I think it just show as a feedback that the commend works